airbyte-integrations/connectors/source-postgres/integration_tests/seed/hook.py

```diff
@@ -114,39 +126,111 @@ def create_table(conn: extensions.connection, schema_name: str, table_name: str)
 def generate_schema_date_with_suffix() -> str:
     current_date = datetime.datetime.now().strftime("%Y%m%d")
     suffix = ''.join(random.choices(string.ascii_lowercase + string.digits, k=8))
-    return f"{current_date}-{suffix}"
+    return f"{current_date}_{suffix}"
 
 def prepare() -> None:
     schema_name = generate_schema_date_with_suffix()
+    print(f"schema_name: {schema_name}")
     with open("./generated_schema.txt", "w") as f:
         f.write(schema_name)
 
 def setup() -> None:
     schema_name = load_schema_name_from_catalog()
     write_supporting_file(schema_name)
+
+def cdc_insert():
+    schema_name = load_schema_name_from_catalog()
+    new_records = [
+        ('4', 'four'),
+        ('5', 'five')
+    ]
+    connection = connect_to_db()
+    table_name = 'id_and_name_cat'
+    if connection:
+        insert_records(connection, schema_name, table_name, new_records)
+        connection.close()
+
+def setup(with_cdc=False):
+    schema_name = load_schema_name_from_catalog()
+    write_supporting_file(schema_name)
     table_name = "id_and_name_cat"
 
+    # Define the records to be inserted
     records = [
         ('1', 'one'),
         ('2', 'two'),
         ('3', 'three')
     ]
 
-    conn = connect_to_db()
+    # Connect to the database
+    connection = connect_to_db()
 
-    if conn:
-        create_schema(conn, schema_name)
-        create_table(conn, schema_name, table_name)
-        insert_records(conn, schema_name, table_name, records)
-        conn.close()
+    if connection:
```

Comment:
This `if` statement shouldn't be needed anymore now that `connect_to_db` always returns a value.